nifi-registry-framework/src/main/resources/db/migration/V2__Initial.sql

```diff
@@ -0,0 +1,58 @@
+-- Licensed to the Apache Software Foundation (ASF) under one or more
+-- contributor license agreements.  See the NOTICE file distributed with
+-- this work for additional information regarding copyright ownership.
+-- The ASF licenses this file to You under the Apache License, Version 2.0
+-- (the "License"); you may not use this file except in compliance with
+-- the License.  You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
```

Comment:
I tried this schema in H2, Postgres and MySQL. For the most part looks good. I did get this error in MySQL:

```
Column length too big for column 'DESCRIPTION' (max = 21845); use BLOB or TEXT instead
```

I did a bit of research and this was the best explanation I found:
https://dev.mysql.com/doc/refman/5.7/en/column-count-limit.html

Seems like 65535 should be fine so long as the total row size stays below that limit. Maybe the 21845 value in the error message is due to a text encoding type (did not dig into it past that). In any case, to maintain mysql compatibility, you may want to lower those max length sizes or change the description fields to TEXT as I don't anticipate we will need to search on those.

Aside from that, this schema looks good to me as a clean starting point for the new database. Nice work.